android/res/values/dimens.xml

```diff
@@ -34,7 +34,7 @@
   <dimen name="height_block_base">48dp</dimen>
   <dimen name="height_item_oneline">56dp</dimen>
   <dimen name="height_item_multiline">72dp</dimen>
-  <dimen name="height_item_edit_bookmark">72dp</dimen>
+  <dimen name="height_item_edit_bookmark">87dp</dimen>
```

Comment:
Simply increasing the max height won't work for even larger fonts. The height should be dynamical.

Reply:
I have successfully made it dynamic now.
Here are some screenshots:
Edit bookmarks:
Before             |  After
:-------------------------:|:-------------------------:
![image](https://user-images.githubusercontent.com/46300244/229342378-f7e7c103-8a44-485d-b377-991efb9be3da.png) | ![image](https://user-images.githubusercontent.com/46300244/229848580-c1c7a8a0-7dec-45c3-b5ed-bd39941b5cc0.png)